gamedata/authors.json

```diff
@@ -20,6 +20,11 @@
       "email": "kyle@iosiro.com",
       "website": "https://github.com/syncikin"
     },
+    "34x4p08": {
+      "name": "Ivan Zakharov",
+      "email": "34x4p08@gmail.com",
+      "website": "https://github.com/34x4p08"
+    },
```

Comment:
You can add a "donate" parameter with an ETH mainnet address to receive donations to your level.

Reply:
Thanks, but I want to miss it.